web-animations-1/Overview.bs

```diff
@@ -1185,9 +1167,10 @@ as CSS Animations [[CSS-ANIMATIONS-1]].
     1.  Cancel that task.
     1.  Set <var>has pending ready promise</var> to true.
 
-1.  If the following three conditions are <em>all</em> satisfied:
+1.  If the following four conditions are <em>all</em> satisfied:
 
     * |animation|'s [=hold time=] is [=unresolved=], and
+    * |animation|'s [=start time=] is [=unresolved=], and
```

Comment:
I'm not quite sure I follow this but I think it probably relates back to my [comment here](https://github.com/w3c/csswg-drafts/issues/2066#issuecomment-598018685).

Reply:
This is required to avoid early exit if start time is set.

Reply:
I'm not sure about this. Won't this mean that calling `play()` on an already running animation makes it become pending? (Since the hold time is unresolved in that case but the start time is not -- previous we would abort the procedure in that case but now we won't.)

Reply:
We could probably fix the above by simply adding some extra flag to this procedure to indicate if we made a change that means that we need to wait for a pending task to run. This might be a more elegant way, however.

Reply:
Added the flag.